docs/getting-started/powerstrip/vagrant-coreos/Vagrantfile

```diff
@@ -42,8 +39,6 @@ Vagrant.configure("2") do |config|
       ip = "172.17.8.#{i+100}"
       host.vm.network :private_network, ip: ip
 
-      #config.vm.provision "docker", images: ["calico/node:#{calico_docker_ver}", "busybox:latest"]
```

Comment:
I've actually uncommented this on my branch so that the calico/node and busybox are preloaded onto the machine during vagrant (which is expected to take time) rather than while running calicoctl node, which could look worse for us if people think it takes a long time to install the container.